js/ui/windowManager.js

```diff
@@ -1693,7 +1999,6 @@ var WindowManager = class {
             Tweener.removeTweens(actor);
             actor.set_scale(1.0, 1.0);
             actor.set_opacity(255);
-            actor.set_pivot_point(0, 0);
```

Comment:
It actually turns out that these changes are not related to wobbly windows, but to flip to hack. Could you please bring them back separately as their original commit? (ceab8c80d8573cdb91b7cdc258b2b5eaf1127f8f)

Reply:
done, a new commit as the original